bin/dev/create_helm_charts_pr.sh

```diff
@@ -59,6 +59,8 @@ sed -i 's@^\(\s\+\)hostname:\s\+".*"$@\1hostname: "registry.suse.com"@' stable/u
 sed -i 's@^\(\s\+\)organization:\s\+".*"$@\1organization: "cap"@' stable/cf/values.yaml
 sed -i 's@^\(\s\+\)organization:\s\+".*"$@\1organization: "cap"@' stable/uaa/values.yaml
 
+sed -i 's@\(\s\+EIRINI_.*_IMAGE:\s\+"registry.suse.com\/\)cap-staging@\1cap@' stable/cf/values.yaml
```

Comment:
This is not correct. The registry name should be stripped completely, e.g. `registry.suse.com/cap-staging/recipe-downloader:0.4.0` should become `recipe-downloader:0.4.0`.

I would also drop the match on `EIRINI...` so this works generally. We should not have any other occurrences of `registry.suse.com/cap-staging/` in our `values.yaml` file.

```suggestion
sed -i 's@registry.suse.com/cap-staging/@@' stable/cf/values.yaml
```

`recipe-downloader:0.4.0` will work correctly because `kube.registry` is set to the correct values both for the staging and for the release chart.

Reply:
+1 to remove the registry part, I didn't realize [this](https://github.com/cloudfoundry-incubator/fissile/blob/develop/kube/pod.go#L516) also applied to env variables. I just looked in the helm chart and it should work: 

```        - name: "EIRINI_DOWNLOADER_IMAGE"
          value: {{if ne (typeOf .Values.env.EIRINI_DOWNLOADER_IMAGE) "<nil>"}}{{if contains "/" .Values.env.EIRINI_DOWNLOADER_IMAGE}}{{.Values.env.EIRINI_DOWNLOADER_IMAGE | quote}}{{else}}{{print .Values.kube.registry.hostname "/" .Values.kube.organization "/" .Values.env.EIRINI_DOWNLOADER_IMAGE | quote}}{{end}}{{else}}""{{end}}
```

Regarding the EIRINI part, I wanted to be explicit because this script assumes that some other script has already copied the images to their final location (in this case the release tool). Since the release tool [explicitly copies only 3 images](https://github.com/SUSE/cloudfoundry/pull/137/files#diff-94defef02205ea151bd44a340383d2aeR161) I wanted to limit the assumption here only to those 3. What I'm thinking is that this script might replace a url for an image which the release tool never actually copied. In any case, there are too many moving parts so none of the solutions is bullet proof. I will go with your suggestion.

One final note, the staging chart doesn't have the "correct" values for the EIRINI images in the staging chart (as described also here: https://github.com/SUSE/cloudfoundry/pull/137/files#r333841514) but does in the released one (after the copying has happened). Since this script prepares the release chart, we can remove the registry part.